src/node/desktop/package.json

```diff
@@ -44,7 +44,7 @@
     "css-loader": "6.5.1",
     "electron": "16.0.7",
     "electron-mocha": "10.1.0",
-    "eslint": "8.2.0",
+    "eslint": "^7.12.1",
```

Comment:
Should using `^` be standard for all other new packages or is it just this case because all the minor updates are expected to be stable?

Reply:
Yes! All minor updates for that one should be stable as there is a Major version already released!